react/src/components/molecules/MainNav/index.js

```diff
@@ -30,10 +30,9 @@ class MainNav extends Component {
   render() {
     return(
       <section className="ma__main-nav">
-        <ul className="ma__main-nav__items js-main-nav">
+        <ul className="ma__main-nav__items js-main-nav" role="menubar">
           {this.props.mainNav.map((item, index) => {
             const buttonId = `button${index}`;
-            const menuId = `menu${index}`;
             const liId = `li${index}`;
             const topItemClasses = ['ma__main-nav__item'];
             const { navSelected } = this.state;
```

Comment:
on line 41, create update is-open and is-closed to react specific classname to enable is-closed to be set to display none on desktop.